12:30:24.357 [main] DEBUG org.springframework.test.context.junit4.SpringJUnit4ClassRunner - SpringJUnit4ClassRunner constructor called with [class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.365 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
12:30:24.374 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
12:30:24.414 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
12:30:24.423 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Neither @ContextConfiguration nor @ContextHierarchy found for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests], using SpringBootContextLoader
12:30:24.428 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfigurationTests-context.xml] does not exist
12:30:24.428 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfigurationTestsContext.groovy] does not exist
12:30:24.429 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
12:30:24.523 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.669 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: using defaults.
12:30:24.669 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
12:30:24.680 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.transaction.TransactionalTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/TransactionDefinition]
12:30:24.681 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/interceptor/TransactionAttribute]
12:30:24.681 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@3e08ff24, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@4d1c005e, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@8462f31, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@24569dba, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@5ddeb7cb, org.springframework.test.context.support.DirtiesContextTestExecutionListener@70ed52de, org.springframework.test.context.event.EventPublishingTestExecutionListener@496bc455, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@59402b8f, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@7188af83, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@6be968ce, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@7c37508a, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@247310d0, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@1033576a]
12:30:24.683 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.684 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.684 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.684 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.691 [main] DEBUG org.springframework.test.context.junit4.SpringJUnit4ClassRunner - SpringJUnit4ClassRunner constructor called with [class org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.691 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
12:30:24.691 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
12:30:24.691 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
12:30:24.692 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Neither @ContextConfiguration nor @ContextHierarchy found for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests], using SpringBootContextLoader
12:30:24.692 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests-context.xml] does not exist
12:30:24.693 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfigurationTestsContext.groovy] does not exist
12:30:24.693 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
12:30:24.696 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.717 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: using defaults.
12:30:24.717 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
12:30:24.718 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.transaction.TransactionalTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/TransactionDefinition]
12:30:24.719 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/interceptor/TransactionAttribute]
12:30:24.719 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@782859e, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@23f5b5dc, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@34bde49d, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@1b1cfb87, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@821330f, org.springframework.test.context.support.DirtiesContextTestExecutionListener@6f43c82, org.springframework.test.context.event.EventPublishingTestExecutionListener@5db6b9cd, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@210ab13f, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@20b2475a, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@7857fe2, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@6f15d60e, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@1be2019a, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@29d80d2b]
12:30:24.720 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.720 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.720 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.720 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.724 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.724 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.724 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.725 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.727 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.727 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.728 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.728 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.728 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.728 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.729 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.735 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.735 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.735 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.735 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.737 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.737 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.737 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.737 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
12:30:24.803 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.803 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.803 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.803 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.807 [main] DEBUG org.springframework.test.context.support.AbstractDirtiesContextTestExecutionListener - Before test class: context [DefaultTestContext@e3c0e40 testClass = KubernetesDiscoveryClientAutoConfigurationTests, testInstance = [null], testMethod = [null], testException = [null], mergedContextConfiguration = [WebMergedContextConfiguration@329dbdbf testClass = KubernetesDiscoveryClientAutoConfigurationTests, locations = '{}', classes = '{class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests$TestConfig}', contextInitializerClasses = '[]', activeProfiles = '{}', propertySourceLocations = '{}', propertySourceProperties = '{spring.cloud.kubernetes.discovery.cacheLoadingTimeoutSeconds=5, spring.cloud.config.enabled=false, spring.cloud.kubernetes.discovery.waitCacheReady=false, org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true, server.port=0}', contextCustomizers = set[org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@185a6e9, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@179ece50, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@2aceadd4, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@2a5c8d3f, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@1ff4931d, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@6200f9cb, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@6a192cfe], resourceBasePath = 'src/main/webapp', contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map['org.springframework.test.context.web.ServletTestExecutionListener.activateListener' -> false]], class annotated with @DirtiesContext [false] with mode [null].
12:30:24.808 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.808 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
12:30:24.813 [main] DEBUG org.springframework.test.context.support.DependencyInjectionTestExecutionListener - Performing dependency injection for test context [[DefaultTestContext@e3c0e40 testClass = KubernetesDiscoveryClientAutoConfigurationTests, testInstance = org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests@43599640, testMethod = [null], testException = [null], mergedContextConfiguration = [WebMergedContextConfiguration@329dbdbf testClass = KubernetesDiscoveryClientAutoConfigurationTests, locations = '{}', classes = '{class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests$TestConfig}', contextInitializerClasses = '[]', activeProfiles = '{}', propertySourceLocations = '{}', propertySourceProperties = '{spring.cloud.kubernetes.discovery.cacheLoadingTimeoutSeconds=5, spring.cloud.config.enabled=false, spring.cloud.kubernetes.discovery.waitCacheReady=false, org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true, server.port=0}', contextCustomizers = set[org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@185a6e9, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@179ece50, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@2aceadd4, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@2a5c8d3f, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@1ff4931d, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@6200f9cb, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@6a192cfe], resourceBasePath = 'src/main/webapp', contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map['org.springframework.test.context.web.ServletTestExecutionListener.activateListener' -> false, 'org.springframework.test.context.event.ApplicationEventsTestExecutionListener.recordApplicationEvents' -> false]]].

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.6.3)

2024-04-21 12:30:25.434  WARN 5985 --- [           main] ubernetesProfileEnvironmentPostProcessor : Not running inside kubernetes. Skipping 'kubernetes' profile activation.
2024-04-21 12:30:25.447  INFO 5985 --- [           main] tesDiscoveryClientAutoConfigurationTests : Starting KubernetesDiscoveryClientAutoConfigurationTests using Java 1.8.0_292 on 003cc2912fe6 with PID 5985 (started by idflakies in /home/idflakies/spring-cloud/spring-cloud-kubernetes)
2024-04-21 12:30:25.447  INFO 5985 --- [           main] tesDiscoveryClientAutoConfigurationTests : No active profile set, falling back to default profiles: default
2024-04-21 12:30:26.806  INFO 5985 --- [           main] org.eclipse.jetty.util.log               : Logging initialized @3369ms to org.eclipse.jetty.util.log.Slf4jLog
2024-04-21 12:30:26.921  INFO 5985 --- [           main] org.eclipse.jetty.server.Server          : jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10
2024-04-21 12:30:26.940  INFO 5985 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@63cf9de0{/__admin,null,AVAILABLE}
2024-04-21 12:30:26.942  INFO 5985 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@5e9f73b{/,null,AVAILABLE}
2024-04-21 12:30:26.959  INFO 5985 --- [           main] o.e.jetty.server.AbstractConnector       : Started NetworkTrafficServerConnector@1e236278{HTTP/1.1, (http/1.1)}{0.0.0.0:38193}
2024-04-21 12:30:26.960  INFO 5985 --- [           main] org.eclipse.jetty.server.Server          : Started @3523ms
2024-04-21 12:30:27.209  INFO 5985 --- [tp1298707825-22] o.e.j.s.handler.ContextHandler.__admin   : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
2024-04-21 12:30:27.447  INFO 5985 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'kubernetesDiscoveryClientAutoConfigurationTests.TestConfig' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2024-04-21 12:30:27.572  INFO 5985 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=c4d006af-a204-306e-a414-6c511b278167
2024-04-21 12:30:27.676  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:27.708  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration' of type [org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:27.714  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:27.719  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'loadBalancerClientsDefaultsMappingsProvider' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration$$Lambda$446/219387406] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:27.725  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'defaultsBindHandlerAdvisor' of type [org.springframework.cloud.commons.config.DefaultsBindHandlerAdvisor] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:27.732  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'kubernetes.manifests-io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:28.058  INFO 5985 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2024-04-21 12:30:28.072  INFO 5985 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-04-21 12:30:28.072  INFO 5985 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.56]
2024-04-21 12:30:28.282  INFO 5985 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-04-21 12:30:28.282  INFO 5985 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2811 ms
2024-04-21 12:30:28.547  INFO 5985 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-04-21 12:30:28.547  INFO 5985 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-04-21 12:30:28.547  INFO 5985 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-04-21 12:30:28.547  INFO 5985 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-04-21 12:30:28.550  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:28.550  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:28.587  INFO 5985 --- [tp1298707825-15] o.e.j.s.handler.ContextHandler.ROOT      : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
2024-04-21 12:30:28.652 ERROR 5985 --- [tp1298707825-16] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:28.652 ERROR 5985 --- [tp1298707825-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:28.674 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:28.675 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:29.546  INFO 5985 --- [pool-9-thread-1] .k.c.d.KubernetesInformerDiscoveryClient : Waiting for the cache of informers to be fully loaded..
2024-04-21 12:30:29.546  INFO 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Cache fully loaded (total 0 services) , discovery client is now available
2024-04-21 12:30:29.676  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:29.676  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:29.687 ERROR 5985 --- [tp1298707825-16] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:29.689 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:29.691 ERROR 5985 --- [tp1298707825-15] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:29.695 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:30.672  INFO 5985 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 34851 (http) with context path ''
2024-04-21 12:30:30.689  INFO 5985 --- [           main] tesDiscoveryClientAutoConfigurationTests : Started KubernetesDiscoveryClientAutoConfigurationTests in 5.85 seconds (JVM running for 7.253)
2024-04-21 12:30:30.690  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:30.696  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:30.697 ERROR 5985 --- [tp1298707825-20] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:30.700 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:30.703 ERROR 5985 --- [tp1298707825-16] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:30.706 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]


  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.6.3)

2024-04-21 12:30:31.168  WARN 5985 --- [           main] ubernetesProfileEnvironmentPostProcessor : Not running inside kubernetes. Skipping 'kubernetes' profile activation.
2024-04-21 12:30:31.169  INFO 5985 --- [           main] iveDiscoveryClientAutoConfigurationTests : Starting KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests using Java 1.8.0_292 on 003cc2912fe6 with PID 5985 (started by idflakies in /home/idflakies/spring-cloud/spring-cloud-kubernetes)
2024-04-21 12:30:31.169  INFO 5985 --- [           main] iveDiscoveryClientAutoConfigurationTests : No active profile set, falling back to default profiles: default
2024-04-21 12:30:31.370  INFO 5985 --- [           main] org.eclipse.jetty.server.Server          : jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10
2024-04-21 12:30:31.372  INFO 5985 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@2bb3f39e{/__admin,null,AVAILABLE}
2024-04-21 12:30:31.373  INFO 5985 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@11f3400c{/,null,AVAILABLE}
2024-04-21 12:30:31.374  INFO 5985 --- [           main] o.e.jetty.server.AbstractConnector       : Started NetworkTrafficServerConnector@12166229{HTTP/1.1, (http/1.1)}{0.0.0.0:46037}
2024-04-21 12:30:31.374  INFO 5985 --- [           main] org.eclipse.jetty.server.Server          : Started @7938ms
2024-04-21 12:30:31.379  INFO 5985 --- [qtp599808137-52] o.e.j.s.handler.ContextHandler.__admin   : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
2024-04-21 12:30:31.429  INFO 5985 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'kubernetesInformerReactiveDiscoveryClientAutoConfigurationTests.TestConfig' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2024-04-21 12:30:31.434  INFO 5985 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=d7232dc4-7a46-3e4e-9978-4dacfc5f6386
2024-04-21 12:30:31.463  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:31.477  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration' of type [org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:31.480  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:31.482  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'loadBalancerClientsDefaultsMappingsProvider' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration$$Lambda$446/219387406] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:31.485  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'defaultsBindHandlerAdvisor' of type [org.springframework.cloud.commons.config.DefaultsBindHandlerAdvisor] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:31.487  INFO 5985 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'kubernetes.manifests-io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-04-21 12:30:31.574  INFO 5985 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2024-04-21 12:30:31.575  INFO 5985 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-04-21 12:30:31.575  INFO 5985 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.56]
2024-04-21 12:30:31.701  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:31.707  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:31.708 ERROR 5985 --- [tp1298707825-20] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:31.710 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:31.718 ERROR 5985 --- [tp1298707825-22] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:31.720 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:31.922  INFO 5985 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-04-21 12:30:31.922  INFO 5985 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-04-21 12:30:31.922  INFO 5985 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-04-21 12:30:31.922  INFO 5985 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-04-21 12:30:31.923  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:31.923  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:31.925  INFO 5985 --- [qtp599808137-45] o.e.j.s.handler.ContextHandler.ROOT      : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
2024-04-21 12:30:31.945 ERROR 5985 --- [qtp599808137-50] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:31.945 ERROR 5985 --- [qtp599808137-45] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:31.948 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:31.948 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:32.711  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:32.717 ERROR 5985 --- [tp1298707825-20] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:32.719 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:32.721  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:32.729 ERROR 5985 --- [tp1298707825-22] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:32.731 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:32.923  INFO 5985 --- [ool-27-thread-1] .k.c.d.KubernetesInformerDiscoveryClient : Waiting for the cache of informers to be fully loaded..
2024-04-21 12:30:32.923  INFO 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Cache fully loaded (total 0 services) , discovery client is now available
2024-04-21 12:30:32.949  INFO 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-04-21 12:30:32.949  INFO 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-04-21 12:30:32.955 ERROR 5985 --- [qtp599808137-45] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:32.955 ERROR 5985 --- [qtp599808137-51] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-04-21 12:30:32.957 ERROR 5985 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:32.957 ERROR 5985 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-04-21 12:30:33.153  INFO 5985 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 44285 (http) with context path ''
2024-04-21 12:30:33.158  INFO 5985 --- [           main] iveDiscoveryClientAutoConfigurationTests : Started KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests in 2.023 seconds (JVM running for 9.721)
2024-04-21 12:30:33.187  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-04-21 12:30:33.187  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-04-21 12:30:33.188  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-04-21 12:30:33.188  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
2024-04-21 12:30:33.189  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-04-21 12:30:33.189  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-04-21 12:30:33.189  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-04-21 12:30:33.189  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
2024-04-21 12:30:33.200  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-04-21 12:30:33.201  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-04-21 12:30:33.201  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-04-21 12:30:33.201  WARN 5985 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
